src/package-utils.ts

```diff
@@ -18,16 +18,25 @@ export enum ManifestDependencyFieldNames {
 }
 
 export enum ManifestFieldNames {
+  Engines = 'engines',
   Name = 'name',
   Private = 'private',
   Version = 'version',
   Workspaces = 'workspaces',
 }
 
+export enum EngineNames {
```

Comment:
These should be named as singular. Not sure about renaming `ManifestFieldNames` in this PR, cause that would be a breaking change, but we can at least properly name `EngineNames`.

```suggestion
export enum ManifestFieldName {
  Engines = 'engines',
  Name = 'name',
  Private = 'private',
  Version = 'version',
  Workspaces = 'workspaces',
}

export enum EngineName {
```

Reply:
Agreed - I'm kind of in favor of internal consistency here though (having `ManifestFieldNames` but `EngineName` can make it easy to mix up and frustrating from a devex perspective). If that makes sense to you, how about we keep them both plural for now and then an upcoming PR can address renaming for style adherence and global consistency?

Not a strong opinion though, feel free to add your suggestion here as a commit to this PR if you're not convinced ^^

Reply:
> If that makes sense to you, how about we keep them both plural for now and then an upcoming PR can address renaming for style adherence and global consistency?

Sounds good to me!